docs/howto/custom-management-commands.txt

```diff
@@ -191,6 +191,17 @@ Attributes
 All attributes can be set in your derived class and can be used in
 :class:`BaseCommand`’s :ref:`subclasses<ref-basecommand-subclasses>`.
 
+.. attribute:: BaseCommand.suppressed_base_arguments
+
+    A set object with the default options entries, which are suppressed.
+    Use the option names you want to suppress the help output for, eg
+    '--verbosity'.
```

Comment:
How about:

> The default command options to suppress in the help output. This should be a set of option names (e.g. `--verbosity`).

Reply:
Fair enough. Will include in the next push.